public/app/features/inspector/InspectErrorTab.tsx

```diff
@@ -35,15 +32,39 @@ export const InspectErrorTab = ({ error }: InspectErrorTabProps) => {
   if (error.message) {
     const { msg, json } = parseErrorMessage(error.message);
     if (!json) {
-      return <div>{msg}</div>;
+      return (
+        <>
+          {error.status && <>Status: {error.status}. Message: </>}
+          {msg}
+        </>
+      );
     } else {
       return (
         <>
           {msg !== '' && <h3>{msg}</h3>}
+          {error.status && <>Status: {error.status}</>}
           <JSONFormatter json={json} open={5} />
         </>
       );
     }
   }
   return <JSONFormatter json={error} open={2} />;
+}
+
+export const InspectErrorTab = ({ errors }: InspectErrorTabProps) => {
+  if (!errors?.length) {
+    return null;
+  }
+  if (errors.length === 1) {
+    return renderError(errors[0]);
```

Comment:
What's the reason behind rendering a single error slightly differently (not wrapping it with an `<Alert>`)?

Reply:
I didn't want to affect the current style since for one error it's clear that it's a single thing. For multiple errors, the best way I found to encasulate them separately was to show different Alerts. Happy to suggestions if there is something else I could try!